### PR TITLE
Fix build fails when built on macOS and Linux

### PR DESCRIPTION
**Pull request by acid-chicken**
Adding some nodes to the project files.

### CODE REVIEW

File: src/Discord.Net.Core/Discord.Net.Core.csproj [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+        <Description>The core components for the Discord.Net library.</Description>
+        <TargetFrameworks>net45;netstandard1.1;netstandard1.3</TargetFrameworks>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <AssemblyName>Discord.Net.Core</AssemblyName>
+        <RootNamespace>Discord</RootNamespace>
+        <Description>The core components for the Discord.Net library.</Description>
+        <TargetFrameworks>netstandard1.1;netstandard1.3</TargetFrameworks>
```
Comment (Still34) [inline src/Discord.Net.Core/Discord.Net.Core.csproj]: Couldn't we just change the `TargetFrameworks` and leave the rest out of the condition? You're duplicating the `AssemblyName` and such and isn't very clean.

Comment (acid-chicken) [reply src/Discord.Net.Core/Discord.Net.Core.csproj]: @Still34 Duplicating elements isn't clean indeed, but, unfortunately, the `When` element can have only one of `Choose`, `ItemGroup` and `PropertyGroup` elements as a child. Would you like to specify these properties with the `Condition` attributes?

Comment (Still34) [reply src/Discord.Net.Core/Discord.Net.Core.csproj]: Yes, we should be able to just use something like
```xml
  <PropertyGroup>
    <AssemblyName>Discord.Net.Core</AssemblyName>
    <RootNamespace>Discord</RootNamespace>
    <Description>The core components for the Discord.Net library.</Description>
    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net45;netstandard1.1;netstandard1.3</TargetFrameworks>
    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.1;netstandard1.3</TargetFrameworks>
  </PropertyGroup>
```

File: src/Discord.Net.Core/Discord.Net.Core.csproj [hunk 2]
```diff
@@ -1,14 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../Discord.Net.targets" />
-  <PropertyGroup>
-    <AssemblyName>Discord.Net.Core</AssemblyName>
-    <RootNamespace>Discord</RootNamespace>
-    <Description>The core components for the Discord.Net library.</Description>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.3</TargetFrameworks>
-  </PropertyGroup>
+  <Choose>
+    <When Condition=" '$(OS)' == 'Windows_NT' ">
```
Comment (Joe4evr) [inline src/Discord.Net.Core/Discord.Net.Core.csproj]: Why even do this in the first place? #RemoveNet45Support

Comment (Still34) [reply src/Discord.Net.Core/Discord.Net.Core.csproj]: One can dream.

File: src/Discord.Net.Core/Discord.Net.Core.csproj (hunk 1 above)
Comment (acid-chicken) [reply src/Discord.Net.Core/Discord.Net.Core.csproj]: @Still34 Fixed in 319bbcffad16e856fd23ffa19ea1c93b9324af7b

**Pull request merged**